lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -24,6 +24,10 @@ class Main < CC::Engine::Analyzers::Base
 
           def process_file(file)
             RubyParser.new.process(File.binread(file), file, TIMEOUT)
+          rescue Timeout::Error
```

Comment:
I'm in favor of rescuing these (I thought we already did: maybe we used to & it got removed?), but if we rescue it for ruby we should probably rescue it for all languages. So this rescue is probably better off in `AnalyzerBase#run`?